vars/debian10.yml

```diff
@@ -0,0 +1,9 @@
+---
+pacemaker_packages:
+ - pcs
+ - pacemaker
+pacemaker_remote_packages:
+ - pcs
+ - pacemaker-remote
+
+cluster_configure_fence_xvm: false
```

Comment:
(low) could you please also add `cluster_firewall: false` to be consistent with description in README?

Reply:
Yes It should be done now

Reply:
ok, looks good.